### ISSUE TITLE
M> Prizes in RHOC for new 'round number' Telegram users

### ISSUE THREAD

**Issue by kitblake** [rchain/Members#182]
When we congratulated the 100th user on joining the rchain_coop group he asked, "What do i get?". The response was: 100 RHOCs. So we sent the guy 100 RHOCs. Now the group is already at 190 members. It would be fun to continue the prizes with the next round number levels: 200, 300, 400, 500, 600, 700, 800, 900, up to 1000. 

That will require a budget of 5500 RHOCs (including the initial 100 prize). Do you like this idea? Or do you think it's silly. You can vote by assigning a budget (of 5500 RHOCs, not Dollars).

**Comment by lapin7** [comment]
I think the reward makes sense when those tg users sign up also for being a member.
And then become a member of Github\rchain\member
And then for creating an issue
And then for adding a budget to an issue
And then for adding rewards to the collaborators on an issue
And also for bringing a friend to RChain

**Comment by kitblake** [comment]
Ok, the Telegram group member number is climbing steadily. When the 300th user arrives, we'll inform him/her that IF s/he becomes a Member, THEN a prize of 300 RHOCs will be awarded. That's like $60 at the private sale rate, so it covers the $20+fee expense of joining.

**Comment by lapin7** [comment]
I suggest to add a bounty of 50 RHOC:
Condition: a Telegram user who is also a Member get's 50 RHOC when s/he
brings in a new Telegram user

Another bounty of 100 RHOC:
Condition: a Telegram user who is also a Member get's 100 RHOC when s/he
brings in a new Github/Member collaborator

Another bounty of 200 RHOC:
Condition: a Telegram user who is also a Member get's 200 RHOC when s/he
brings in a new Github/Rchain developer



--
Cheers,
HJ

**Comment by kitblake** [comment]
The reward campaign seems to be working. We informed the 300th Telegram member that, if he becomes a Member of the Co-op, he'll get a 300 RHOC reward. He checked it out and said he'd join. 

We need to get the transaction logistics worked out. I would suggest that we create a wallet for the reward payouts, share the private key with a couple operators and ops, transfer 5500 RHOCs into it, and dole out the rewards from there. All trackable on the blockchain. We might want a spreadsheet to store details.

**Comment by lapin7** [comment]
For coop-payments there's already a multisig Gnosis wallet. I can just
enter the payment to the winning address. And Kenny can approve. Then the
payment will be send.
A few month a go, I could do it on my own. With the PRE-member
registration. People send in about $20 worth of BTC or ETH for the member
fee. And they got a 100 RHOC in return. It went into the accounting books
and all was fine.

--
Cheers,
HJ

**Comment by kitblake** [comment]
Sending payments from the Gnosis wallet won't have the reaction speed that we'd prefer for the Telegram environment, but since the rewards require Member registration and KYC verification that will create a slower process anyway. 

So: once the new milestone Member has been verified, a Telegram operator will collect and pass the payment address to Ops. Then, when the payout has been sent, the operator will "make some noise" by publicly informing the Telegram user that the RHOCs have arrived. This is the workflow:
- A milestone Telegram user joins the group (400th, 500th, etc).
- An operator informs the milestone user of the potential for a reward (if the user becomes a Member of the Co-op). 
- If/when the user becomes a verified Member, an operator will ask the user to provide an ETH address in a PM (best if the operator PMs the user first so the address doesn't happen to get posted in the channel).
- The operator passes the Member name, ETH address, and amount to Ops.
- When the payout has been made, Ops informs the Telegram operator.
- The operator doublechecks the ETH address. 
- The operator announces the payout in the group with a @mention of the milestone user.

**Comment by lapin7** [comment]
OK, then the gnosis wallet will work like a back up and reimburse at the
end of the month to the payer of milestone rewards.

--
Cheers,
HJ

**Comment by kitblake** [comment]
That's a good idea. Plus it means Ops doesn't take on additional tasks of dealing with reward logistics. The Telegram operators will track and handle that.

**Comment by Mervyn853** [comment]
Telegram just reached 700 members and I think it has become an essential tool for RChain membership Coop onboarding.

Had some discussion with @kitblake and would like to suggest doing away with the round number prizes for membership onboarding as some of the recent winners have come on solely with the motivation to receive RHOCs.

I would like to suggest an alternative rewards program where we reward those who actively answer questions posed by the general public when the Telegram Moderators/Admin are unavailable to respond. 

Invited a bot that can help track community health / engagement and active members and the data can be found here : [https://combot.org/chat/-1001123521235] RChain Combot

**Comment by ResonanceXX** [comment]
Hello Merv, I  was a round number myself, but with membership of coop and contributor status on here. The telegram group growth has been phenomenal. 
I would still support any other kind of reward, this is because human beings thrive more when expectations are held on to. Any new ideas as to engaging members is always welcomed. And I shall be lurking, always aiming to win some. Cheers! 

Sent from Yahoo Mail on Android 
 
  On Wed, 3 Jan 2018 at 7:22, Mervyn853<notifications@github.com> wrote:   
Telegram just reached 700 members and I think it has become an essential tool for RChain membership Coop onboarding.

Had some discussion with @kitblake and would like to suggest doing away with the round number prizes for membership onboarding as some of the recent winners have come on solely with the motivation to receive RHOCs.

I would like to suggest an alternative rewards program where we reward those who actively answer questions posed by the general public when the Telegram Moderators/Admin are unavailable to respond.

Invited a bot that can help track community health / engagement and active members and the data can be found here : [https://combot.org/chat/-1001123521235] RChain Combot

**Comment by kitblake** [comment]
I'm pleased to report that our last round number prize winner, #400, is not in it for the money. He's a Java/Python dev and will go after some bounties. However as @Mervyn853 pointed out, it's likely that we'll get users who join for free money, especially with the skyrocketing RHOC price. 

So to sum up, we had 100, 200, 300, and 400 RHOC winners for a budget of 1000 RHOCs. 

Now we have to figure out how to reward active and helpful Telegram users. Any ideas @ResonanceXX? You're a self-confessed lurker in the Tg group :)

This week @patrick727 offered an on-the-spot reward in Telegram to the guy who reported the RChain website not loading:
```
Patrick M, [02.01.18 19:21]
whoever caught that please shoot me a pm with your RHOC address
```
He got thumbs up from users for that. Way to go.

**Comment by kitblake** [comment]
One of the more active 'helpers' on Telegram is YSGjay. He answers lots of questions and provides links. He's on Discord but not on Github yet. He should get some RHOCs. The question is how.

**Comment by ResonanceXX** [comment]
Kitblake, well said. I lurk mainly because of several limitations on my end. I am confined to the use of my mobile phone to make contributions to the co-operative. Once this is resolved, then my recent contribution would be insignificant to my future contribution.
The rewards system could be split in a way that is consistent with equity and fairness. Also people who show dedicated response to issues affecting the coop need to feel valued. 
Cheers! 

Sent from Yahoo Mail on Android 
 
  On Fri, 5 Jan 2018 at 20:37, kitblake<notifications@github.com> wrote:   
I'm pleased to report that our last round number prize winner, #400, is not in it for the money. He's a Java/Python dev and will go after some bounties. However as @Mervyn853 pointed out, it's likely that we'll get users who join for free money, especially with the skyrocketing RHOC price.

So to sum up, we had 100, 200, 300, and 400 RHOC winners for a budget of 1000 RHOCs.

Now we have to figure out how to reward active and helpful Telegram users. Any ideas @ResonanceXX? You're a self-confessed lurker in the Tg group :)

This week @patrick727 offered an on-the-spot reward in Telegram to the guy who reported the RChain website not loading:
Patrick M, [02.01.18 19:21]
whoever caught that please shoot me a pm with your RHOC address

He got thumbs up from users for that. Way to go.

**Comment by Ojimadu** [comment]
I also feel we should do away with the TG sign up bounty, current winners can receive their reward. I still don't understand how the combot aggregates it data that might be a way to reward TG users but that's not until we understand how the bot gathers its data and sets its parameters.
I won't like us to set rewarrds for activity as that might breed something we may not like and some users would start engaging in non-constructive chats just to increase their activity. BTW YSGjay has been helpful and a reward won't be bad but how?

**Comment by kitblake** [comment]
Yes, I've been puzzling about the 'how' too, and it was YSGjay's activity that triggered it. :)

**Comment by lapin7** [comment]
I've PM'd YSGjay for getting in touch for a reward.

**Comment by kitblake** [comment]
Great, ysgjay is coming onboard. Just invited him to join this repo. 
"Awaiting jplepel's response"..

**Comment by lapin7** [comment]
I've created #223  >M Extremely helpful Telegram interaction
<https://github.com/rchain/Members/issues/223>

And set the budget to $500 :-)

--
Cheers,
HJ

**Comment by kitblake** [comment]
Then I suggest we close this issue and pick up the 'how' in #223. 
However, I need budget on this issue for january. Does the invoice generator pick up closed issues? If not we'll wait till the end of the month.

**Comment by ResonanceXX** [comment]
@Kitblake I support that position. 

Sent from Yahoo Mail on Android 
 
  On Wed, 10 Jan 2018 at 17:39, kitblake<notifications@github.com> wrote:   
Then I suggest we close this issue and pick up the 'how' in #223.
However, I need budget on this issue for january. Does the invoice generator pick up closed issues? If not we'll wait till the end of the month.

**Comment by lapin7** [comment]
The budget for #223 will stay, even if it's closed.
The new rule will be that issues without a budget and older then 1 month,
will be closed. And they won't appear anymore in "Budget 201802" nor in
"Rewards 201802".


--
Cheers,
HJ

**Comment by kitblake** [comment]
Ok, closing this.
